paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -291,8 +278,15 @@ internal class CustomerSheetViewModel @Inject constructor(
     }
 
     private fun onBackPressed() {
-        val shouldExit = backStack.value.singleOrNull() != null ||
-            viewState.value is CustomerSheetViewState.SelectPaymentMethod
+        val currentBackStack = backStack.value
+        val shouldExit = currentBackStack.isEmpty() ||
```

Comment:
I’m a bit confused by the need for this complicated condition. Can’t we just decide with `currentBackStack.size == 1`?